README.rst

```diff
@@ -40,6 +41,8 @@ Supported formats
 * xls - Uses Python ``xlrd`` module.
 * xlsx - Uses Python ``xlrd`` module.
 * zip - Uses Python ``zipfile`` module.
+* hwp - Uses Python ``pyhwp`` module as CLI tool.
+* jpg, jpeg, png, bmp, gif - Use ``/usr/bin/tesseract`` CLI tool and ``pytesseract`` module.
```

Comment:
I think this is actually part of your other pull request (for OCR).

Reply:
Yes, I worked with crossing branches, I think I was so confused.